dist/lux/bootstrap.css

```diff
@@ -5251,9 +5251,9 @@ input[type="button"].btn-block {
 }
 
 .alert-primary {
-  color: #0e0e0e;
-  background-color: #d1d1d1;
-  border-color: #bfbfbf;
+	color: #004085;
+	background-color: #cce5ff;
+	border-color: #b8daff;
```

Comment:
To be consistent to the rest of the codebase, would you please indent your code with 2 spaces instead of tabs

Reply:
Ok thanks, works!

Reply:
ok but and the scss file ?

Reply:
Scss has no conflicts.